cdap-watchdog/src/main/java/co/cask/cdap/logging/save/LogSaver.java

```diff
@@ -135,17 +141,19 @@ private void createProcessors(Set<Integer> partitions) throws Exception {
     }
   }
 
-  private void cancelLogCollectorCallbacks() {
+  private Set<Cancellable> cancelLogCollectorCallbacks() {
```

Comment:
better to rename this method since it not longer cancels the callbacks but just gets set of cancellables?